index.html

```diff
@@ -104,53 +105,79 @@ <h1>Plug and play</h1>
             </p>
-            <p>
-                We can guide you through technical tradeoffs, and help you find new ways to increase cash flow for your product.
+            <p class="quotee mb-0">Isaac, Product Owner</p>
+        </section>
+        <div class="solution">
+            <h2 class="mt-0">[our solution]</h2>
+            Our consultants take on that exact perspective, to take on the role as temporary
+            CTO if you don't have one, or the role of directly supporting your CTO in the important
+            work they are doing.
+            <a href="">Check it out <span>↬</span></a>
+        </div>
+        <div class="icon icon-bottom arrow-2">⟿</div>
+    </div>
+    <div class="segment">
+        <div class="icon icon-top arrow-3">⮍</div>
+        <div class="problem">
+            <h2 class="mt-0">Getting the team you actually need</h2>
+            Hiring and training a development team requires a continuous, large investment.
+            If developers leave, you lose the knowledge you've spent time building. Under constrained
+            budgets, it's impossible to cover all the skills needed in a development team. Alternatively,
+            consulting companies work on a project basis and aren't there to make adjustments as you
+            need them, and disagreements about scope often arise.
+        </div>
+        <section class="customer-quote-section">
+            <p class="quote-text my-0">
+                There has been good communication and dedi&shy;cation from deranged throughout the partner&shy;ship. [...] We have a close client-supplier relation&shy;ship. I see deranged as being my own in-house developers. That is how it feels.
             </p>
-            <div class="read-more">Get awesome</div>
-        </a>
-        <a href="/no-restrictions/" class="product product-no-resctrictions">
-            <img src="/img/terrain-icon.svg">
-            <h1>A path towards a system that doesn't restrict you</h1>
-            <p>
-                If the software you're currently using is starting to seem more restricting than enabling, we can help you plan a constructive way forward.
-                We will help you get back in control of the technological heart of your business, and give you the freedom to adapt to changing business practices.
+            <p class="quotee mb-0">Ninna Toft Hansen, then-COO</p>
+        </section>
+        <div class="solution">
+            <h2 class="mt-0">[our solution]</h2>
+            We have an experienced team that is used to working together. We integrate closely
+            with your business as if we were your own developers, but we hit the ground running on,
+            and you only pay for us in the periods where you need development. We're a plug and play
+            team: plug us into your business and we will start delivering value.
+            <a href="">Check it out <span>⇝</span></a>
```

Comment:
De her `span`s skal altså helst ikke være uden `class` på -- det er dårlig stil at style spans generelt når der burde være en beskrivende class for hvad det er for en ting og hvorfor den skal styles.